### PR TITLE
MCU boot for pca20035

### PR DESCRIPTION
**Pull request by joakimtoe**

### CODE REVIEW

**Comment by nvlsianpu** [comment]
Please follow ncs commit message guideline. I expect something like here: https://github.com/NordicPlayground/fw-nrfconnect-mcuboot/commit/8b0e50cd5898e66e8bf92b5378c5b29fb0838471

File: boot/zephyr/boards/nrf52840_pca20035.conf [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Enable RTT logging
```
Comment (mbolivar) [inline boot/zephyr/boards/nrf52840_pca20035.conf]: I'm confused. The next line (`CONFIG_LOG=n`) isn't enabling logging; it's disabling logging.

File: boot/zephyr/CMakeLists.txt [hunk 2]
```diff
@@ -54,6 +54,8 @@ macro(app_set_runner_args)
   endif()
 endmacro()
 
+include($ENV{ZEPHYR_BASE}/../nrf/cmake/boilerplate.cmake NO_POLICY_SCOPE)
```
Comment (mbolivar) [inline boot/zephyr/CMakeLists.txt]: Can we get a comment about why this line is needed? I'm guessing it's because it's an out of tree board definition?

**Comment by nvlsianpu** [comment]
@joakimtoe Can you rebate your changes at current master?

**Pull request closed without merging**